rules/70-joystick.rules

```diff
@@ -0,0 +1,13 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="joystick_end"
+ENV{ID_INPUT}=="", GOTO="joystick_end"
+ENV{ID_INPUT_JOYSTICK}=="", GOTO="joystick_end"
```

Comment:
nitpick, if ID_INPUT_JOYSTICK is set but not ID_INPUT then that's a bug elsewhere, you can drop the ID_INPUT check